astroquery/alma/tests/test_alma_remote.py

```diff
@@ -131,13 +131,14 @@ def test_data_proprietary(self, alma):
         query = "select top 1 member_ous_uid from ivoa.obscore where " \
                 "obs_release_date > '{}'".format(now)
         result = alma.query_tap(query)
-        assert len(result.table) == 1
+        assert len(result) == 1
         # proprietary
         assert alma.is_proprietary(result.table[0][0])
         # non existent
         with pytest.raises(AttributeError):
             alma.is_proprietary('uid://NON/EXI/STING')
 
+    @pytest.mark.skipif("SKIP_SLOW")
```

Comment:
It would be better to use the slow marker introduced in `pytest-astropy` version 0.10.0. See also #2420.